Core/include/Acts/EventData/MultiTrajectory.hpp

```diff
@@ -191,51 +217,104 @@ class TrackStateProxy {
   template <bool RO = ReadOnly, bool ReadOnlyOther,
             typename = std::enable_if<!RO>>
   void copyFrom(const TrackStateProxy<M, ReadOnlyOther>& other,
-                TrackStatePropMask mask = TrackStatePropMask::All) {
+                TrackStatePropMask mask = TrackStatePropMask::All,
+                bool onlyAllocated = true) {
```

Comment:
Maybe you should add the `onlyAllocated` to the parameter list.

Reply:
Done.